install/kubernetes/microcks/templates/secret.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if and (.Values.ingresses) (not .Values.microcks.ingressSecretRef) (.Values.microcks.generateCert) }}
```

Comment:
Secret for GRPC is needed even if we do not expose the ingresses because it is used directly by the GRPC server within the Spring Boot app. I think it would be better to introduce a new variable `microcks.grpcSecretRef` and just have:

```
{{- if hasKey  .Values.microcks.grpcSecretRef) }}
```

Also, this new secret ref must be used in `deployment.yaml` where we mount the GRPC secret into the main pod.

Reply:
I've added microcks.grpcSecretRef

Reply:
Awesome! Will have a look at it in a few hours.